sdk/core/core-client/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.0.1 (Unreleased)
 
+- [Breaking] If the response body is empty and the mapper for it says it is nullable, then a null is returned.
```

Comment:
Feels like we should make this `1.1.0` since the response is changing

Reply:
Done.